src/main/resources/data.sql

```diff
@@ -1,9 +1,31 @@
-USE mmotodo;
+-- Insert sample data into the 'tag' table
+INSERT INTO tag (name, description) VALUES
+                                        ('Work', 'Tasks related to work'),
+                                        ('Personal', 'Personal tasks'),
+                                        ('Urgent', 'Tasks that need immediate attention');
```

Comment:
The .sql files are not executed when using flyway

Reply:
We can remove these files completely. I made a new issue about this. #34
Merge this code into #29 that deals with initialization of the database with some default values at startup.